index.html

```diff
@@ -1,23 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<link rel="reset" type="text/css" href="css/reset.css">
+		<link href='<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>'>
```

Comment:
you've got a little issue with this link tag. The href attribute has an entire other `<link>` tag inside, which is not exactly what you want. Instead, you only need the code you copied from google fonts:

```
<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
```